atomic_reactor/plugins/pre_koji_parent.py

```diff
@@ -209,7 +215,7 @@ def manifest_list_entries_match(self, image, build_id):
 
         archives = self.koji_session.listArchives(build_id)
         koji_archives_data = {}
-        for archive in archives:
+        for archive in [a for a in archives if a['btype'] == 'image']:
```

Comment:
We could use a generator expression here?

Reply:
And also can we use constant? Something like `KOJI_BTYPE_IMAGE` as we have for operator-manifests

Reply:
Done

Reply:
Still says `'image'` I think.

Reply:
my page was outdated when I commented this. I did not realize there was a second request :)

On it